contentScript.js

```diff
@@ -351,6 +358,11 @@ if (window.location.hash) {
 var current_href = location.href;
 setInterval(function () {
   if (current_href !== location.href) {
+    while(alreadySetIntervalsForEnableRefresh.length>0){
```

Comment:
@sushrut111 Did you notice multiple intervals existing?

Reply:
Yes I did.
But the line you are commenting on is to deregister all intervals on page change.

Reply:
Yup, that's why I asked if you noticed multiple and that's why you were popping them